### ISSUE TITLE
integration not working with HA

### ISSUE THREAD

**Issue by prague34** [SmartArduino/DoHome#17]
No 'version' key in the manifest file for custom integration 'dohome'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'dohome'

integration not working with HA

**Comment by prague34** [comment]
When discovered a device ,

Incorrect HomeKit code. Please check it and try again.

**Comment by HoGC** [comment]
It's fixed, you can try it
https://github.com/SmartArduino/DoHome/tree/master/DoHome_HassAssistant_Component

**Comment by prague34** [comment]
> It's fixed, you can try it
> 
> https://github.com/SmartArduino/DoHome/tree/master/DoHome_HassAssistant_Component

Thanks , yes it's fixed. But something behave anormal like brightness , even if you set the brightness to 100%, it goes back to 39% and some of the colors are not working properly. 
![image](https://user-images.githubusercontent.com/26159033/130370072-af8c4a69-a370-4247-ae73-7e5bd99540e3.jpeg)

**Comment by Dima-Kal** [comment]
> > It's fixed, you can try it
> > https://github.com/SmartArduino/DoHome/tree/master/DoHome_HassAssistant_Component
> 
> Thanks , yes it's fixed. But something behave anormal like brightness , even if you set the brightness to 100%, it goes back to 39% and some of the colors are not working properly. ![image](https://user-images.githubusercontent.com/26159033/130370072-af8c4a69-a370-4247-ae73-7e5bd99540e3.jpeg)

Yes im having the same issue, always sayd 39%, have you managed to find a solution for that?
